deployments/observability.mdx

```diff
@@ -104,3 +104,18 @@ Which would provide you with an interactive terminal.
 
 For more options please take a look at our
 [Readme](/reference/cli/#architect-exec-resource-flags----command).
+
+## Port forwarding
+
+Sometimes you want to access services running inside your cluster without exposing them publicly.
+The `port-forward` command will allow you access these services on your local machine.
+This can be useful for exposing non-http services for debugging.
+
+```sh
+$ architect port-forward -a my_account -e my_environment react-app.services.api --port 5000 --target-port 8080
```

Comment:
@tjhiggins based on this comment, I assume that you would need a component that is already deployed on this environment? if so, we'd need to note that as a requirement to using port-forwarding

Reply:
would this be in lieu of `architect dev`?

Reply:
architect port-forward would be used in tandem with architect deploy. Say you deploy a postgres service and you wanted to connect to it from pgadmin on your localhost. You could expose it on your localhost from the k8s cluster. 

https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/